Library/Formula/ansible-cmdb.rb

```diff
@@ -16,11 +16,13 @@ class AnsibleCmdb < Formula
 
   def install
     bin.mkpath
+    man1.mkpath
     inreplace "Makefile" do |s|
       s.gsub! "/usr/local/lib/${PROG}", prefix
       s.gsub! "/usr/local/bin", bin
+      s.gsub! "/usr/local/share/man/man1/", man1 + 'ansible-cmdb.1'
+      s.gsub! "mandb", "#mandb"
```

Comment:
Both PRs fboender/ansible-cmdb#49 and fboender/ansible-cmdb#50 have been merged upstream already. So we can get rid of these two lines again as soon as the next ansible-cmdb version will be released …